### ISSUE TITLE
ESLINT errors down to zero!

### ISSUE THREAD

**Issue by alejandronanez** [curiositylab/curiosity#43]
We can do this gradually!

**Comment by mubaris** [comment]
Yeah, it should be done.

**Comment by alejandronanez** [comment]
I think it would be nice to get rid of aaaaall the eslint errors (I think we have a TON). But doing that in one PR maybe too much. I'm ok if we do this gradually, one PR per file or something like that!

For this to be a thing, we need #17 to be merged in `V2` first. BTW

**Comment by mubaris** [comment]
@alejandronanez Since I'm having some issues, #17 might take more time to finish. I'll try to send a PR as soon as I can.

**Comment by alejandronanez** [comment]
@mubaris yeah don't feel rushed about it. I just had this on my mind and wanna make sure I don't forget!

**Comment by mubaris** [comment]
Cool

**Comment by jearl4** [comment]
Hope y'all don't mind if I ran eslint and cleaned up a few of the errors. Have a PR out there with the changes and will start work on the emoji file later.

**Comment by Suneet-M** [comment]
Are all the errors resolved? Or can I contribute as well?

**Comment by paulopacitti** [comment]
is this issue still open? Can I contribute?

**Comment by devcer** [comment]
I would like to work on this. Can you assign this to me?

**Comment by mehargiri** [comment]
Is this issue still open? I want to contribute, can you assign this to me?

**Comment by ASP124** [comment]
Hello, Is this issue still open to be resolved or can it be assigned to me?

**Comment by whotterre** [comment]
Hi there! Is this issue still open or assignable to me?
